### PR TITLE
Update docker.rb for zap

### PR DESCRIPTION
**Pull request by nuomi1**
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Update docker.rb for zap

### CODE REVIEW

File: Casks/docker.rb [hunk 1]
```diff
@@ -11,4 +11,17 @@
   auto_updates true
 
   app 'Docker.app'
+
+  zap delete: [
+                '/Library/LaunchDaemons/com.docker.vmnetd.plist',
```
Comment (jawshooah) [inline Casks/docker.rb]: Please add `uninstall launchctl: 'com.docker.vmnetd'` and remove this line.

**Review by jawshooah — CHANGES_REQUESTED** [review]

**Review by jawshooah — APPROVED** [review]

File: Casks/docker.rb [hunk 2]
```diff
@@ -11,4 +11,18 @@
...
+
+  uninstall launchctl: 'com.docker.vmnetd'
+
+  zap delete: [
+                '/Library/PrivilegedHelperTools/com.docker.vmnetd',
+                '~/Library/Application Scripts/com.docker.helper',
+                '~/Library/Caches/KSCrashReports/Docker',
+                '~/Library/Caches/com.docker.docker',
+                '~/Library/Caches/com.plausiblelabs.crashreporter.data/com.docker.docker',
+                '~/Library/Containers/com.docker.docke',
```
Comment (jawshooah) [inline Casks/docker.rb]: Is `com.docker.docke` a typo?

Comment (nuomi1) [inline Casks/docker.rb]: I am sorry.

**Comment by jawshooah** [comment]
Thanks!

**Pull request merged**